conda/recipes/libcuspatial/meta.yaml

```diff
@@ -34,6 +35,8 @@ build:
 requirements:
   build:
     - cmake >=3.12.4
+    - gtest
+    - gmock
```

Comment:
Are these needed even if we're not building tests?

Reply:
I think we need these in CI, because we're not using `CPMFindPackage()` for `GTest` anymore, we're using the `cudf::gtest` target now.

If I understand correctly, this is the chain of events:

* `libcudf` is installed via conda
* cuspatial does `CPMFindPackage(cudf)`, which finds cuDF via `find_package(cudf COMPONENTS testing)`
* `cudf-config.cmake` tries to do `find_package(GTest)` because we asked it for the `testing` component
* If `GTest` isn't installed via conda, `find_package(cudf)` will fail

The correct place to add these is `libcudf/meta.yml`, but I added them here a few days ago to see if it would unblock CI.

Reply:
Added in https://github.com/rapidsai/cudf/pull/7734/commits/b53d6f4a9ea6faba5b21147f747dfde6f4d285ae